scripts/projectView.js

```diff
@@ -0,0 +1,32 @@
+var projectView={};
+
+//projectView.titleList = function() {
+  //$('article').each(function() {
+
+
+projectView.navBar = function() {
+  $('#nav-bar').on('click', 'li', function() {
+    $('.tab-content').hide();
+    $('#' + $(this).data('content')).fadeIn();
+  });
+
+  $('#nav-bar.tab:first').click();
```

Comment:
You need to have a space between selectors. Right now the click event isn't firing on the element you're intending to click.